mypy.ini

```diff
@@ -7,6 +7,9 @@ ignore_missing_imports = True
 [mypy-clickhouse_driver]
 ignore_missing_imports = True
 
+[mypy-clickhouse_driver.errors]
```

Comment:
@fpacifici I was getting missing imports errors for this when I ran it without `--ignore-missing-imports` so that's why I thought I needed it.